cli/cmd/tapRunner.go

```diff
@@ -420,7 +420,7 @@ func watchApiServerEvents(ctx context.Context, kubernetesProvider *kubernetes.Pr
 
 			event, err := wEvent.ToEvent()
 			if err != nil {
-				logger.Log.Errorf(fmt.Sprintf("Error parsing Mizu resource event: %+v", err))
+				logger.Log.Debugf(fmt.Sprintf("Error parsing Mizu resource event: %+v", err))
```

Comment:
don't we need to add continue after this?
and i think fmt.Sprintf can be removed